pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -141,27 +141,6 @@ public void testCaseWhenStatements() {
     Assert.assertEquals(caseFunc.getOperands().get(6).getLiteral().getFieldValue(), 0L);
   }
 
-  @Test(expectedExceptions = SqlCompilationException.class)
-  public void testInvalidCaseWhenStatements() {
```

Comment:
Couldn't we reuse the text to verify this works now?

Even something as simple as compiling the same SQL query and assert no failure is found. May not be super useful, but given it is already written, we could use it

Reply:
Done.